app/src/main/res/values-es/strings.xml

```diff
@@ -301,25 +351,29 @@
   <string name="logs_view_button">Ver registros…</string>
   <string name="logs_view_with">Ver registros con…</string>
   <string name="logs_share_button">Compartir registros…</string>
-  <string name="logs_share_subject">Registros de la aplicación Phoenix </string>
+  <string name="logs_share_subject">Registros de la aplicación Phoenix</string>
   <string name="logs_share_title">Compartir registros de Phoenix…</string>
 
   <!-- //////////////// payment details //////////////// -->
 
   <string name="paymentdetails_loading">Recuperando detalles de pago…</string>
   <string name="paymentdetails_error">Ocurrió un error al recuperar los detalles de este pago.\n\nPor favor inténtalo de nuevo.</string>
+
   <string name="paymentdetails_status_sent_successful"><![CDATA[<b>Enviado</b> %1$s]]></string>
   <string name="paymentdetails_status_sent_pending"><![CDATA[<b>Pendiente…</b>]]></string>
-  <string name="paymentdetails_status_sent_failed"><![CDATA[Error <b>de pago</b>.<br />No se ha enviado dinero.]]></string>
+  <string name="paymentdetails_status_sent_failed"><![CDATA[<b>Fallo</b> en el pago.<br />No se ha enviado dinero.]]></string>
+
   <string name="paymentdetails_status_received_pending">El pago aún no se ha recibido.</string>
   <string name="paymentdetails_status_received_successful"><![CDATA[<b>Recibido</b> %1$s]]></string>
+
   <string name="paymentdetails_desc_label">Desc.</string>
-  <string name="paymentdetails_destination_label">Enviar a</string>
-  <string name="paymentdetails_fees_label">Comisión</string>
+  <string name="paymentdetails_destination_label">Enviado a</string>
+  <string name="paymentdetails_fees_label">Comisiones</string>
```

Comment:
destination_label, fees_label, desc_label are being used both in the sending and the payment details screens:

![image](https://user-images.githubusercontent.com/1238928/88079662-a9c92300-cb43-11ea-8f39-36954bd8b132.png)

So `Enviado a` is not a good fit for the _sending_ screen, and `Enviar a` is not a perfect fit for the _details screen_. I personally think that the sending screen is more important than the details one, so If I had to choose, I would prefer `Enviar a` over `Enviado a`.

What do you think about `Destino`? I think it is not a perfect label, but anyway a good fit for both screens.

Reply:
"Destino" is very good for this! Done.